ev-dev-tools/src/ev_cli/error_parsing.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 - 2022 Pionix GmbH and Contributors to EVerest
```

Comment:
we should probably change the date to 2023 or (even better) remove the date range alltogether